package.json

```diff
@@ -23,6 +23,7 @@
     "@docusaurus/remark-plugin-npm2yarn": "^3.1.0",
     "@ionic-docs/preset-classic": "^1.1.0",
     "clsx": "^2.0.0",
+    "docusaurus-theme-github-codeblock": "^2.0.2",
```

Comment:
What happens if we build the site without an internet connection? e.g. What's the failure mode look like if I'm doing a local build and don't have wifi access?

Reply:
Is there any way it could store both the information needed to re-fetch the lines from github and a fallback right in the markdown? That would make it possible to build offline without too much trouble I think. I don't know too much about how docusaurus plugins are structured to know if there would be flexibility to do something like this.

Reply:
I haven't been able to look into this yet but I hope Docusaurus allows to pull the code as part of the server side generation process.